app/readme.md

```diff
@@ -81,6 +81,12 @@ This will take a moment, after which you can go to <http://localhost:3000/>, and
 
 ### I'm having difficulty with setup or coding
 
+#### seeing errors regarding migrations for db
+
+delete the postgres docker image
+delete the postgres folder inside app/data
+re-run `docker-compose up --build `
```

Comment:
Without dot points this will have messed up formatting. Maybe change to:

```
* Stop postgres docker container
* Delete the `postgres` folder in `app/data`
* Re-run `docker-compose up --build`